gohelper_test.go

```diff
@@ -158,6 +158,46 @@ func TestCredentialsForMissingRelationshipErrrors(t *testing.T) {
 	}
 }
 
+func TestCredentialsForRelationshipPropertyQueryStructDefinedReturns(t *testing.T) {
+	config, err := psh.NewRuntimeConfigReal(helper.RuntimeEnv(psh.EnvList{}), "PLATFORM_")
+	helper.Ok(t, err)
+
+	creds, err := config.Credentials("database")
+	helper.Ok(t, err)
+
+	helper.Equals(t, true, creds.Query.IsMaster)
+}
+
+func TestCredentialsForRelationshipPropertyQueryAbsentReturns(t *testing.T) {
+	config, err := psh.NewRuntimeConfigReal(helper.RuntimeEnv(psh.EnvList{}), "PLATFORM_")
```

Comment:
I wonder if these tests would be more valid if they took an existing config and mutated it, rather than having them lying around as discrete test data?  I'm honestly unsure here, nor do I know how much work that would be.